doc/k8s/sqlflowhub/Dockerfile

```diff
@@ -1,8 +1,14 @@
 FROM sqlflow/sqlflow
 
+# Installed npm
+RUN apt-get -y install npm
+
 RUN mkdir -p /etc/jhub && \
     bash -c "pip install jupyterhub && \
     pip install notebook"
+    
+# Added to get rid of the of the Proxy Config Folder not founf Errror resulting in POD crash    
+RUN npm install -g configurable-http-proxy
```

Comment:
Can use one `RUN` line to install `npm` and `configurable-http-proxy`:

``` bash
RUN apt-get -y install npm && \
    npm install -g configurable-http-proxy
```

Ref [Dockerfile Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices) :
> Split long or complex RUN statements on multiple lines separated with backslashes to make your Dockerfile more readable, understandable, and maintainable.